CMakeLists.txt

```diff
@@ -3,13 +3,26 @@
 #
 cmake_minimum_required (VERSION 3.8)
 
-find_program(OPEN_CPP_COVERAGE "OpenCppCoverage")
-
-if(OPEN_CPP_COVERAGE)
-    MESSAGE(STATUS "OpenCppCoverage -- found")
-else(OPEN_CPP_COVERAGE)
-    MESSAGE(STATUS "OpenCppCoverage -- not found")
-endif(OPEN_CPP_COVERAGE)
+IF(WIN32)
+    find_program(OPEN_CPP_COVERAGE "OpenCppCoverage")
+    if(OPEN_CPP_COVERAGE)
+        MESSAGE(STATUS "OpenCppCoverage -- found")
+    else(OPEN_CPP_COVERAGE)
+        MESSAGE(STATUS "OpenCppCoverage -- not found")
+    endif(OPEN_CPP_COVERAGE)
+ENDIF(WIN32)
+IF(UNIX)
```

Comment:
should this be an else if with an else for an unsupported warning?

Reply:
well I wasnt planning to throw not supported messages as you can read the readme